changelogs/fragments/pr_1315.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - zabbix-agent Fix zabbix_agent_logfile if condition #1314
```

Comment:
need to insert a " - " after zabbix_agent (also it should be zabbix_agent).  You can remove the ticket number also, it gets included automagically when we merge it.